src/main/webapp/i18n/de/exam.json

```diff
@@ -25,7 +25,9 @@
         },
         "examParticipation": {
             "connected": "Verbunden. Ihre Antworten werden alle 60s gespeichert.",
-            "disconnected": "Nicht verbunden. Gehen Sie online, um Ihre Änderungen auf dem Server zu speichern, aber Sie können weiterarbeiten!"
+            "disconnected": "Nicht verbunden. Gehen Sie online, um Ihre Änderungen auf dem Server zu speichern, aber Sie können weiterarbeiten!",
```

Comment:
I just checked a few jsons and Artemis usually "dutzes" the students, so I would consider using "Du" here instead of "Sie".